web/src/components/UploadAirgapBundle.jsx

```diff
@@ -313,37 +314,9 @@ class UploadAirgapBundle extends React.Component {
     }, 1000);
   };
 
-  getSupportBundleCommand = async (slug) => {
-    const res = await fetch(
-      `${process.env.API_ENDPOINT}/troubleshoot/app/${slug}/supportbundlecommand`,
```

Comment:
Should we preserve this call and have this copy updated in the troubleshoot app?

Reply:
@rianfowler That might be a good idea. Who would reach out to?

Reply:
I would just tag the requester and jason on the ticket. something like "we can hardcode this on the FE but it looks like it's currently getting it from a troubleshoot api."

Reply:
I think this makes sense since the response will have the correct troubleshoot command.